cernopendata/modules/theme/webpack.py

```diff
@@ -27,21 +27,22 @@
 from invenio_assets.webpack import WebpackThemeBundle
 
 
-js = WebpackThemeBundle(
```

Comment:
Question: why we got rid of the JS bundle? How is e.g. `popper.js` being loaded?

Reply:
It is now included inside the `theme` bundle (I am basically following the recommended way in cookiecutter invenio instance, see [here](https://github.com/inveniosoftware/cookiecutter-invenio-instance/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/%7B%7Bcookiecutter.package_name%7D%7D/theme/webpack.py)).

The usage ends up being `{{ webpack['cernopendata_css.css'] }}` for CSS and `{{webpack['cernopendata_js.js'] }}` for JS, both being picked from the [`entry`](https://github.com/cernopendata/opendata.cern.ch/pull/3000/files#diff-8cbc780ffa24ccb6fdbb0958a7ce997f655647976c428e0d25d19ebe80d17e19L36) dictionary.

Reply:
That I understand. But I don't see a JavaScript entry bundle, `cernopendata_js` has been deleted if I'm not wrong.

Reply:
When I meant now I meant with the force push, `cernopendata_js.js` was indeed missing, thanks for spotting.

Reply:
OK, I see that [you've just added it](https://github.com/cernopendata/opendata.cern.ch/compare/503f0ab7c4126fad19760609b6cf5222768f9a7c..3ebe5f5804860cd9f75a4770d8a4e068844f6ff5) back...